project/Build.scala

```diff
@@ -16,7 +16,7 @@ object B extends Build {
   override lazy val settings = super.settings ++ Seq(
     organization := "com.typesafe.genjavadoc",
     version := "0.9-SNAPSHOT",
-    scalaVersion := "2.10.4",
+    scalaVersion := "2.11.5",
```

Comment:
Yes, it was just that we had not change in akka yet.
How is the branching working here? Should master be on 2.11 or 2.10?